Python/hbayesdm/base.py

```diff
@@ -597,24 +597,32 @@ def _designate_stan_model(self, model: str) -> StanModel:
         """
         stan_files = os.path.join(_common, 'stan_files')
         model_path = os.path.join(stan_files, model + '.stan')
-        cache_file = 'cached-%s-hbayesdm=%s-pystan=%s.pkl' % (
-            model, _hbayesdm_version, _pystan_version)
-        try:
-            with open(cache_file, 'rb') as cached_stan_model:
-                sm = pickle.load(cached_stan_model)
-            with open(model_path, 'r') as model_stan_code:
-                assert sm.model_code == model_stan_code.read()
-        except (FileNotFoundError, AssertionError):
-            sm = StanModel(
-                file=model_path, model_name=model, include_paths=[stan_files])
-            with open(cache_file, 'wb') as f:
-                pickle.dump(sm, f)
-        except:  # All other exceptions
-            raise RuntimeError(
-                'Cache file is corrupted. Please remove file `' +
-                cache_file + '` and run again.')
+        cache_file = 'cached-%s-hbayesdm=%s-pystan=%s.pkl' % \
+            (model, _hbayesdm_version, _pystan_version)
+
+        if os.path.exists(cache_file):
+            try:
+                with open(cache_file, 'rb') as cached_stan_model:
+                    sm = pickle.load(cached_stan_model)
+                with open(model_path, 'r') as model_stan_code:
+                    assert sm.model_code == model_stan_code.read()
+                does_exist = True
+            except Exception:
+                print('Invalid cached StanModel:', cache_file)
+                print('Remove the cached model...')
```

Comment:
I think there'd be less confusion to tell the user: `Removing existing cache model to create a new one...` or `Rewriting cache model...` or sth like that, so as not to confuse user to have to manually "remove the cached model"